### DISCUSSION TITLE
why save and cancel for QPopupEdit aren't working

### DISCUSSION THREAD

**Discussion by freemant2000** [elimintz/justpy#381] [General]
Hi,

I see that this is mentioned in the doc and actually found support for these events in the source code. Just wondering why this is not working? Is it a problem in Quasar, Vue or justpy itself?

Many thanks!

**Comment by elimintz** [comment]
I don't know. The way to solve this problem is to look at the source code of Quasar and try figuring out why the interaction with justpy fails.
